spec/features/revisions_spec.rb

```diff
@@ -6,7 +6,7 @@
   describe 'when the record has 2 revisions' do
     let(:record) do
       with_versioning do
-        create(:issue)
+        create(:issue, node_id: current_project.issue_library.id)
```

Comment:
Why is this needed? The Issues factory already accounts for the right Node association.